client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableRow/InvestigationTableRow.tsx

```diff
@@ -220,6 +220,13 @@ const InvestigationTableRow = ({
                                 <Visibility />
                             </IconButton>
                         </Tooltip>
+                        {indexedRow.wasAbroad &&
```

Comment:
why wasAbroad changes are here? I think you didnt pull changes before starting to work on the bug.